workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/carousel-card/index.css

```diff
@@ -0,0 +1,122 @@
+.carousel-card {
+	background: var(--color-neutral-1, #F7F7F8);
+	border-radius: 0.75rem;
+	height: auto;  
+	padding: 2.5rem;
+	width: 100%;
+}
+
+.carousel-card-container {
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+}
+
+.carousel-card-information {
+	display: flex;
+	flex-direction: column;
+	gap: 1.5rem;
+	height: 100%;
+	justify-content: space-between;
+}
+
+.card-cta {
+	align-items: center;
+	cursor: pointer;
+	display: flex;
+	flex-direction: row;
+	gap: 0.5rem;
+
+	a {
+		color: var(--color-brand-primary, #005BFF);
+		font-size: 1.25rem;
+		font-weight: 600;
+		text-decoration: none;
+	}
+
+	.cta-icon {
+		transition: transform 0.3s ease-in-out; 
+	}
+}
+
+.card-cta:hover .cta-icon {
+	transform: translateX(4px); 
+}
+
+.card-description {
```

Comment:
Since `card-description` is a generic class, it’s best to add it within a specific scope to avoid affecting other components.